packages/mobile/src/web3/contracts.ts

```diff
@@ -155,7 +184,7 @@ export function* getWeb3() {
 }
 
 // Used for cases where the kit's web3 must be accessed outside a saga
-export async function getWeb3Async() {
+export async function getWeb3Async(): Promise<Web3> {
   const kit = await getContractKitAsync()
-  return kit.web3
+  return kit?.web3!
```

Comment:
Seems dangerous! Are we sure it can't never be `undefined`?

Reply:
You good point and it is dangerous. The danger was there before as well but after I refactored some stuff I helped typescript see the danger as well. I'll massage things upstream and add some more checks to remove the need for this.